www/src/components/guidelines/color/modal.js

```diff
@@ -138,7 +151,16 @@ const modalContent = (palette, color) => {
       const textColor = getTextColor(c.contrast)
 
       colors.push(
-        <Box width="100%" display={{ lg: `flex` }} key={`palette-${index}`}>
+        <Box
+          sx={{
+            display: `block`,
```

Comment:
Not sure, but do we need the explicit `display: block` for `Box` (https://theme-ui.com/components/box makes it look like this renders as a `div` by default, which has `display: block`)?